README.md

```diff
@@ -0,0 +1,79 @@
+gceimgutils
```

Comment:
Preference: I like having dash in names to make it more readable. It does mean typing the command is harder though. E.g. gce-img-utils.

Reply:
Seems we end up one way or the other as the existing ec2imgutils has no dashes.

Reply:
Right and this is general preference, I tend to name packages with dashes and modules with underscores. :shrug: